docs/third-party-developers/extensibility/data-store/validation.md

```diff
@@ -38,73 +39,40 @@ An example of how the data should be structured:
 
 When the checkout process begins, it will check if this data store has any entries, and if so, it will stop the checkout process from proceeding. It will also show any errors that are hidden. Setting an error to hidden will not clear it from the data store!
 
-## Selectors
-
-### getValidationError
-
-Returns the validation error.
-
-#### _Parameters_
+## Usage
 
--   _errorId_ `string` - The error ID to get validation errors for.
-
-#### Example
+To utilize this store you will import the `CART_STORE_KEY` in any module referencing it. Assuming `@woocommerce/block-data` is registered as an external pointing to `wc.wcBlocksData` you can import the key via:
 
 ```js
-const store = select( 'wc/store/validation' );
-const billingFirstNameError = store.getValidationError( 'billing-first-name' );
+import { CART_STORE_KEY } from '@woocommerce/block-data';
```

Comment:
This is the wrong key, it should be `VALIDATION_STORE_KEY`.

Reply:
Good catch. I've corrected the wrong key.